robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -67,13 +67,19 @@ def _is_current_page(self):
         """ Verify we are on the Recurring Donations Detail page
             by verifying that the url contains '/view'
         """
-        self.selenium.location_should_contain(
-            "/lightning/r/npe03__Recurring_Donation__c/",
-            message="Current page is not a Recurring Donations record view",
-        )
-        locator = npsp_lex_locators["bge"]["button"].format("Edit")
-        edit_button = self.selenium.get_webelement(locator)
-        self.selenium.wait_until_page_contains_element(edit_button, error="Recurring donations Details page did not load fully")
+        for i in range(3):
```

Comment:
curious, so this button is sometimes not visible on initial load?

Reply:
yes , the button does not load all the times.